testing/tests/DevExpress.ui.widgets.dataGrid/filtering.integration.tests.js

```diff
@@ -396,9 +396,9 @@ QUnit.module('Initialization', baseModuleConfig, () => {
             }]
         });
 
-        assert.equal(calculateFilterExpressionCallCount, 6, 'calculateFilterExpression call count');
+        assert.equal(calculateFilterExpressionCallCount, 5, 'calculateFilterExpression call count');
```

Comment:
because now in dataController, we call _calculateAdditionalFilter after check of isColumnsDefined, call count has decreased.

Ticket of this test (T802967), is about calculateFilterExpression being called infinite times, so this change is fine.